tests/by-util/test_tail.rs

```diff
@@ -1119,13 +1092,13 @@ fn test_invalid_num() {
         .fails()
         .stderr_str()
         .starts_with("tail: invalid number of lines: '1024R'");
-    #[cfg(not(target_pointer_width = "128"))]
+
     new_ucmd!()
         .args(&["-c", "1Y", "emptyfile.txt"])
         .fails()
         .stderr_str()
         .starts_with("tail: invalid number of bytes: '1Y': Value too large for defined data type");
-    #[cfg(not(target_pointer_width = "128"))]
```

Comment:
Feels like we should definitely keep these? CI can't cover every possible arch and OS we want to run the tests on

Reply:
ok